tests/test_pipeline.py

```diff
@@ -1,19 +1,24 @@
 from pathlib import Path
 import pytest
 from phaser import Pipeline, Phase, batch_step
-from steps import adds_row
 
 
 current_path = Path(__file__).parent
 
 
 # LMDTODO: Move more pipeline tests out of test_basics and call that test_phase
 def test_number_go_up(tmpdir):
+    @batch_step
```

Comment:
I had this step in a library of test steps, but I trust there was a reason to move it into the test

Reply:
There was.  The file that is being used -- `departments.csv` -- changed structure (the columns changed from `id` and `name` to `key` and `value`), and I did not want to couple that file to the steps that would be shared with several tests.  It seemed like too fragile and an unnecessary coupling.

Reply:
Yes it might have been too optimistic to make a bunch of reusable steps for tests.  no worries.